src/Kdyby/Doctrine/EntityManager.php

```diff
@@ -281,6 +284,88 @@ public function onDaoCreate(EntityManager $em, Doctrine\Common\Persistence\Objec
 
 
 
+	/****************** Kdyby\Persistence\QueryExecutor *****************/
+
+
+
+	/**
+	 * Create a new QueryBuilder instance that is pre-populated for this entity name
+	 *
+	 * @param  string|NULL  $alias
+	 * @param  string|NULL  $indexBy
+	 * @param  string|NULL  $entityName
+	 * @return \Kdyby\Doctrine\DqlSelection
+	 */
+	public function select($alias = NULL, $indexBy = NULL, $entityName = NULL)
```

Comment:
Yeah that is right. Having `$alias` and `$indexBy` in `EntityManager::createQueryBuilder` isnt exactly clear either but it is required by the interface.